### ISSUE TITLE
Getting all the binaries from a package

### ISSUE THREAD

**Issue by energizerhub** [Julian/venvs#67]
It would be nice to have `--link-all` get all the binaries from a package. Here's pipsi's take: https://github.com/mitsuhiko/pipsi/blob/master/pipsi/scripts/find_scripts.py

**Comment by Julian** [comment]
Sounds reasonable, though ech that script...

Needs a decent syntax for config files too.

**Comment by Julian** [comment]
Maybe after #63, the latter is just `link = ["*"]`.

**Comment by altendky** [comment]
Minus python/python2/python3/pip/...?

**Comment by Julian** [comment]
Yeah you'd ignore those presumably in globs unless they were specifically asked for.
